src/main/webapp/forms/noarray.gndlink.jsp

```diff
@@ -8,7 +8,7 @@
             if (schemaOrgProperty != null && !schemaOrgProperty.isEmpty()) {
                 schemaOrgAttribute = " property=\"" + schemaOrgProperty + "\"";
             }
-            String link = "<a href=\"https://d-nb.info/gnd/" + DBtoHTML(gndId) + "\" target=\"_blank\" style=\"display: inline-block; line-height: 40px; vertical-align: middle; margin-top: 10px;\"" + schemaOrgAttribute + "> " + gndIcon + "</a>";
+            String link = "<a href=\"https://d-nb.info/gnd/" + DBtoHTML(gndId) + "\" target=\"_blank\" style=\"display: inline-block; vertical-align: middle; margin-bottom: -8px; margin-top: -5px;\"" + schemaOrgAttribute + "> " + gndIcon + "</a>";
```

Comment:
A negative margin is always a dirty solution, because if the surrounding objects change you have to change this as well. Is there a better solution?

Reply:
i removed the minus margins, the field is now a little bigger but still not too big and if a name hase two lines or more it won`t get a problem with the margins